### ISSUE TITLE
Feature request: file transfer

### ISSUE THREAD

**Issue by geckolinux** [jitsi/jitsi-meet#534]
Hello, would it be possible to add an option to Jitsi Meet to share files between participants? It should be fairly straightforward, since the WebRTC standard does support it. Example implementation [here](https://www.webrtc-experiment.com/RTCMultiConnection/group-file-sharing-plus-text-chat.html).

**Comment by ZuzooVn** [comment]
Are there any update here?

**Comment by saghul** [comment]
Nope, nothing to report. We are not currently working on this feature.

**Comment by ZuzooVn** [comment]
@saghul : Thanks for your information.
Do you have any suggestion for community?

**Comment by saghul** [comment]
@ZuzooVn suggestions as in what to do? Not really. If you really want to see this happen I suggest you come up with a (thorough) design on how you plan on implementing this and bring it up here for discussion.

**Comment by petervnv** [comment]
Hi guys,

Is there any news on this front? Anyone working on this?
File transfer is a very useful feature although I can understand it is not so easy to implement in jitsi-meet.

Cheers,
Peter

**Comment by damencho** [comment]
Nothing had changed since the last comment of Saghul. 
The only thing is that we added it to gsoc projects (dropbox integration), we chose a student, but he/she gave up from the program and this project failed. 
So contributors are welcome.

**Comment by lrq3000** [comment]
Please, we need this feature! Nowadays there are lots of implementations of file transfers using WebRTC, surely something can be done.

**Comment by kik-doctor** [comment]
Hello, jitsi fans.
For all jitsi customers need help with file transferring in jitsi meet.
Please come up with this feature.

**Comment by ebellussi** [comment]
I would like to work on it while I think it should be interesting to implement such a feature to be more competitive against commercial software like Zoom, Teams and Skype. Is anyone else interested?

**Comment by geckolinux** [comment]
@ebellussi Very interested in testing it, thanks! Unfortunately I'm not a developer though.
